packages/material-ui/src/Paper/Paper.js

```diff
@@ -45,6 +46,18 @@ const PaperRoot = experimentalStyled(
     overridesResolver,
   },
 )(({ theme, styleProps }) => {
+  // Inspired by https://github.com/material-components/material-components-ios/blob/bca36107405594d5b7b16265a5b0ed698f85a5ee/components/Elevation/src/UIColor%2BMaterialElevation.m#L61
+  const getOverlayAlpha = (elevation) => {
+    let alphaValue;
+    if (elevation < 1) {
+      alphaValue = 5.11916 * elevation ** 2;
+    } else {
+      alphaValue = 4.5 * Math.log(elevation + 1) + 2;
+    }
+    return (alphaValue / 100).toFixed(2);
+  };
+  const overlayColor = alpha(theme.palette.common.white, getOverlayAlpha(styleProps.elevation));
```

Comment:
This is probably good enough:

```suggestion
  const overlayColor = alpha('#fff', getOverlayAlpha(styleProps.elevation));
```